packages/workbox-google-analytics/build.js

```diff
@@ -25,7 +25,7 @@ const productionBuildConfigs = generateBuildConfigs({
     iife: pkg.main,
   },
   baseDir: __dirname,
-  moduleName: 'goog.offlineGoogleAnalytics',
+  moduleName: 'workbox.offlineGoogleAnalytics',
```

Comment:
The package name for this is now workbox-google-analytics, should this be changed to workbox.googleAnalytics ?